### PR TITLE
Ensure a slash between the authority and the path of a URL

### PR DESCRIPTION
**Pull request by dr0i**
See #279.

### CODE REVIEW

**Comment by fsteeg** [comment]
> I will check how this case is covered by the spec tests when integrating this fix in the 1.1 branch.

To clarify: the fix does not break any of the spec tests for 1.1 or 1.0, so I think we can merge it. What I meant is that I will look into what test case to enable or add in the spec tests to cover the scenario fixed here.

**Comment by dr0i** [comment]
+1, will merge next week unless @ansell objects.

File: core/src/test/java/com/github/jsonldjava/utils/JsonUtilsTest.java [hunk 1]
```diff
@@ -15,6 +17,19 @@
...
+    @Test
+    public void resolveTest() {
+        final String baseUri = "http://mysite.net";
+        final String pathToResolve = "picture.jpg";
+        String resolve = "";
+
+        try {
+            resolve = JsonLdUrl.resolve(baseUri, pathToResolve);
+        } catch (final Exception e) {
+            assertTrue(false);
```
Comment (ansell) [inline core/src/test/java/com/github/jsonldjava/utils/JsonUtilsTest.java]: The usual pattern for this is to have a fail at the end, and put the assertEquals inside of the try-catch block. I know the next method does it this way, but it isn't ideal.

Comment (dr0i) [reply core/src/test/java/com/github/jsonldjava/utils/JsonUtilsTest.java]: @ansell please have a look if  https://github.com/jsonld-java/jsonld-java/pull/280/commits/9359f10b89b9a16b7670ccb0ee0beafb0295e3bd does what you've suggested.

Comment (ansell) [reply core/src/test/java/com/github/jsonldjava/utils/JsonUtilsTest.java]: Looks okay now. I may get to improving the overall style someday, but this is good enough for today.

**Comment by fsteeg** [comment]
> I will look into what test case to enable or add in the spec tests to cover the scenario fixed here

Added tests in https://github.com/w3c/json-ld-api/pull/430.

**Pull request merged**